examples/byte-type/byte_type.bal

```diff
@@ -1,15 +1,15 @@
 import ballerina/io;
 
 public function main() {
-    // Defines a `byte` variable.
+    // Define a `byte` variable.
     byte a = 12;
     io:println(a);
 
-    // Defines a `byte` array using the ordinary array literal.
+    // Define a `byte` array using the ordinary array literal.
```

Comment:
"ordinary" sounds odd in this context doesn't it?

Reply:
Any suggestions? I think what was intended here was that it is the usual array.

Reply:
Use the spec terminology? i.e., list constructor?